app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,41 @@
+<div id="form">
+  <h2>Sign up</h2>
+  <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+    <%= render "devise/shared/error_messages", resource: resource %>
+
+    <div class="field">
+      <%= f.label :name %><br />
+      <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
+    </div>
+
+    <div class="field">
+      <%= f.label :photo %><br />
+      <%= f.text_field :photo, autofocus: true, autocomplete: "photo" %>
+    </div>
+
+    <div class="field">
+      <%= f.label :email %><br />
+      <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+    </div>
+
+    <div class="field">
+      <%= f.label :password %>
+      <% if @minimum_password_length %>
+      <em>(<%= @minimum_password_length %> characters minimum)</em>
+      <% end %><br />
+      <%= f.password_field :password, autocomplete: "new-password" %>
+    </div>
+
+    <div class="field">
+      <%= f.label :password_confirmation %><br />
+      <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+    </div>
+
+    <div class="actions">
+      <%= f.submit "Sign up" %>
+    </div>
+  <% end %>
+
+  <%= render "devise/shared/links" %>
+</div>
```

Comment:
- [ ] You did an excellent job of incorporating style into the sign-up form :point_down: . However, as stated in the requirements, let's make sure it matches the wireframe provided. What you need to change in your styling is to make sure the placeholders are inside the inputs. Like this :point_down: 
```
    <div class="field">
      <%= f.text_field :name, placeholder: 'Name', autofocus: true, autocomplete: "name" %>
    </div>

    <div class="field">
      <%= f.text_field :photo, placeholder: 'Photo', autofocus: true, autocomplete: "photo" %>
    </div>

    <div class="field">
      <%= f.email_field :email, placeholder: 'Email', autofocus: true, autocomplete: "email" %>
    </div>

    <div class="field">
      <% if @minimum_password_length %>
      <em>(<%= @minimum_password_length %> characters minimum)</em>
      <% end %><br />
      <%= f.password_field :password, placeholder: 'Password', autocomplete: "new-password" %>
    </div>

    <div class="field">
      <%= f.password_field :password_confirmation, placeholder: 'Password Confirm', autocomplete: "new-password"%>
    </div>
```

- [ ] And please make sure that the Sign-up button has the same width compared to the inputs. :ok_hand: 

#### Requirement :point_down: 
![Screenshot from 2023-05-19 19-28-14](https://github.com/Chaseknis/blog_app/assets/89228743/a44ac0c5-363e-4b9c-98e9-046b4d813bdb)

#### Your form :point_down: 
![Screenshot from 2023-05-26 23-39-40](https://github.com/Solobarine/whispers/assets/89228743/6aeb2d5b-5176-42de-847a-12589fc9c340)


#### The wireframe :point_down: 
![Screenshot from 2023-05-26 23-41-19](https://github.com/Solobarine/whispers/assets/89228743/6d167b94-8165-4138-96de-729d31742635)